src/components/v2/Tabs/index.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { State } from 'react-powerplug';
+import { ComponentMeta } from '@storybook/react';
+import { withThemeProvider, withCenterStory } from 'stories/decorators';
+import { Tabs } from '.';
+
+export default {
+  title: 'Components/Tabs',
+  component: Tabs,
+  decorators: [withThemeProvider, withCenterStory({ width: 350 })],
+} as ComponentMeta<typeof Tabs>;
+
+export const Default = () => (
+  <State initial={{ activeTabIndex: 0 }}>
```

Comment:
Nice, we have a `withState` decorator but it currently is customized for a checkbox. Would be nice to have a versitile decorator for adding state to stories

Reply:
Yeah I saw that, but I didn't know if you wanted it to be specific to form components so I just imported this library in dev

Reply:
We can keep it as is, I meant this comment mostly as food for thought